webapp/sources/rudder/rudder-core/src/main/scala/com/normation/cfclerk/domain/VariableAndSectionSpec.scala

```diff
@@ -437,3 +436,20 @@ object DisplayPriority {
     }
   }
 }
+
+sealed  trait CompositionRule
+case object WorstReport extends CompositionRule
+case object SumReport extends CompositionRule
+case class ComponentReport(component : String) extends CompositionRule
```

Comment:
Component name, and use the compliance of all reports matching this component name

Like previous comment, now that we do have an id for all method call we should change the reporting to get the id now and not matching component name and value (I'm started a doc on the subject, It will be shared on Slack)